wasmcloud_host/lib/wasmcloud_host/lattice/state_monitor.ex

```diff
@@ -195,6 +195,48 @@ defmodule WasmcloudHost.Lattice.StateMonitor do
     %State{state | providers: providers}
   end
 
+  defp process_event(
+         state,
+         %Cloudevents.Format.V_1_0.Event{
+           data: %{"actors" => _actors, "providers" => _providers},
+           datacontenttype: "application/json",
+           source: _source_host,
+           type: "com.wasmcloud.lattice.host_heartbeat"
+         }
+       ) do
+    Logger.info("Handling host heartbeat")
+    # TODO: handle heartbeats
+    state
+  end
+
+  defp process_event(
+         state,
+         %Cloudevents.Format.V_1_0.Event{
+           data: %{"public_key" => _pkey},
+           datacontenttype: "application/json",
+           source: _source_host,
+           type: "com.wasmcloud.lattice.health_check_passed"
+         }
+       ) do
+    Logger.info("Handling heartbeat success")
```

Comment:
I know that this is just a placeholder, but this isn't a heartbeat, this is a health check.